notebooks/Sigma_test_Notebook.ipynb

```diff
@@ -0,0 +1,358 @@
+{
+ "metadata": {
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.9.2-final"
+  },
+  "orig_nbformat": 2,
+  "kernelspec": {
+   "name": "python392jvsc74a57bd02556566ac8fd219b8b3cb0c76c8e4f226085ddd0e18628d37e0282e6a68fc0ff",
+   "display_name": "Python 3.9.2 64-bit ('.venv')"
+  },
+  "metadata": {
+   "interpreter": {
+    "hash": "2556566ac8fd219b8b3cb0c76c8e4f226085ddd0e18628d37e0282e6a68fc0ff"
+   }
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "source": [
+    "# Sigma example notebook\n",
+    "\n",
+    "This notebook should showcase some use cases with regards to Sigma."
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install -q timesketch_api_client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from timesketch_api_client import search\n",
+    "from picatrix.lib import state as state_lib\n",
+    "\n",
+    "import altair as alt\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from timesketch_api_client import config\n",
+    "from timesketch_import_client import helper\n",
+    "from timesketch_import_client import importer\n",
+    "from timesketch_api_client import search"
+   ]
+  },
+  {
+   "source": [
+    "# Create the sketch and import some data\n",
+    "\n",
+    "First we want to create a sketch and import a small data set. It is recommended to do that with the dev docker container."
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ts_client = config.get_client()"
+   ]
+  },
+  {
+   "source": [
+    "# We create a new sketch\n",
```

Comment:
why is this a header?

![notebook_screenshot](https://user-images.githubusercontent.com/8701947/115109952-58ca4c00-9f68-11eb-95a7-aa8227e91635.png)